### ISSUE TITLE
Dev Edition [Elementor v3.3.0-dev8 Elementor Pro v3.4.0-dev8]: Social Icons appear incorrectly in front-end, fine in editor

### ISSUE THREAD

**Issue by d19dotca** [elementor/elementor#15093]
## Prerequisites
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest development version of Elementor.

## Description
In my "Elementor Dev" testing site, I noticed in the last few dev versions that the Social Icons no longer view correctly. It shows fine in Editor, but not when viewing the page from the front-end. No changes were made by me other than updating to the latest Elementor Dev editions.

Currently running on:
Elementor v3.3.0-dev8
Elementor Pro v3.4.0-dev8

It should look like this (and does from editor view):
<img width="385" alt="Screen Shot 2021-05-25 at 7 08 38 PM" src="https://user-images.githubusercontent.com/8153796/119592834-12011a80-bd8e-11eb-9e66-55c388adf2cc.png">

But unfortunately looks like this in the latest versions on Dev Edition (which is incorrect):
<img width="421" alt="image" src="https://user-images.githubusercontent.com/8153796/119592995-51c80200-bd8e-11eb-88fc-47907bad2e32.png">

## Steps to reproduce
There were no steps performed other than installing the Dev Edition and updating to the latest Dev releases. Here is what I did:

1. Cloned my live website to a new dedicated "Elementor Dev" website for testing purposes.
2. Verified everything looked correct.
3. Installed the Elementor Dev Edition plugin.
4. Updated the versions of Elementor provided by the Dev Edition.
5. Viewed website to make sure it was okay, saw this change to the Social Icons widget.
6. Regenerated CSS files and cleared all cache.
7. Issue still remains.

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [X] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.41 (Ubuntu)
	MySQL version: (Ubuntu) v8.0.23
	PHP Version: 7.4.3
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: https://www.e-dev.d19.ca
	Home URL: https://www.e-dev.d19.ca
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 1024M
	Permalink Structure: /%category%/%postname%/
	Language: en-CA
	Timezone: America/Vancouver
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_CA
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15

== Active Plugins ==
	AuthLDAP
		Version: 2.4.10
		Author: Andreas Heigl <andreas@heigl.org>

	Better Search Replace
		Version: 1.3.4
		Author: Delicious Brains

	Elementor
		Version: 3.3.0-dev8
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.4.0-dev8
		Author: Elementor.com

	MainWP Child
		Version: 4.1.6.1
		Author: MainWP

	Pods - Custom Content Types and Fields
		Version: 2.7.27
		Author: Pods Framework Team

	Redis Object Cache
		Version: 2.0.19-dev
		Author: Till Krüss

	SMTP Mailer
		Version: 1.1.2
		Author: naa986

	ToolKit For Elementor
		Version: 1.3.12
		Author: ToolKit For Elementor


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible
	ToolKit For Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	ToolKit For Elementor: Incompatible

== Elements Usage ==
	
	archive : 1
		archive-posts : 1
		theme-archive-title : 1
	footer : 1
		divider : 1
		heading : 4
		nav-menu : 1
		search-form : 1
		social-icons : 1
		text-editor : 4
	header : 1
		icon-list : 1
		nav-menu : 1
		theme-site-logo : 1
	page : 2
		heading : 4
		spacer : 1
		theme-site-logo : 1
	section : 1
		heading : 1
		posts : 1
	single : 2
		global : 1
		heading : 1
		image : 1
		post-info : 1
		post-navigation : 1
		share-buttons : 1
		text-editor : 1
		theme-post-content : 2
		theme-post-title : 2
	widget : 0
		form : 2
	wp-post : 8
		accordion : 1
		alert : 4
		button : 17
		code-highlight : 2
		counter : 2
		divider : 2
		form : 1
		global : 5
		heading : 51
		icon-box : 7
		icon-list : 6
		image : 1
		lottie : 4
		portfolio : 1
		social-icons : 3
		table-of-contents : 1
		tabs : 3
		testimonial-carousel : 1
		text-editor : 90
		toggle : 1
	wp-page : 13
		accordion : 1
		alert : 1
		button : 13
		call-to-action : 1
		divider : 2
		facebook-page : 1
		global : 1
		heading : 19
		lottie : 1
		menu-anchor : 4
		posts : 3
		price-list : 2
		price-table : 12
		social-icons : 2
		text-editor : 26
		wp-widget-tag_cloud : 1
	single-post : 1
		button : 1
		divider : 1
		heading : 1
		image : 1
		portfolio : 1
		post-info : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-title : 1
	single-page : 1
		shortcode : 1
		table-of-contents : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-title : 1
	search-results : 1
		archive-posts : 1
		theme-archive-title : 1
	error-404 : 1
		heading : 2
		nav-menu : 1
		search-form : 1


== Log ==
	
Log: showing 20 of 432021-05-25 18:59:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:00:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:01:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:02:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:03:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:04:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:05:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:05:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:05:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev7',
)]
2021-05-25 19:05:16 [info] elementor::elementor_updater Started 
2021-05-25 19:05:16 [info] Elementor/Upgrades - _on_each_version Start  
2021-05-25 19:05:17 [info] Elementor/Upgrades - _on_each_version Finished 
2021-05-25 19:05:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev8',
)]
2021-05-25 19:05:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev5',
  'to' => '3.3.0-dev8',
)]
2021-05-25 19:05:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.0-dev6',
  'to' => '3.4.0-dev8',
)]

```
</details>

**Comment by DorShahar** [comment]
Hi @d19dotca 
Thanks for your feedback.
We are aware of this and we are acting to fix it soon.

Therefore - I merged this issue with  #13122, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!

**Comment by d19dotca** [comment]
Sorry, I wasn't aware this was the same. The ticket you referenced seems to be more around alignment of social icons, where-as mine was the actual display ofd the icons of them, not the alignment. But if they're all going to be resolved in the same PR, then that's great. :-)
